programs/token-metadata/program/tests/resize.rs

```diff
@@ -1119,23 +887,12 @@ mod resize {
         .await
         .unwrap();
 
-        let edition = match token_standard {
-            TokenStandard::NonFungible | TokenStandard::ProgrammableNonFungible => {
-                assert_before_master_edition(&mut context, nft.edition.unwrap()).await;
-                nft.edition.unwrap()
-            }
-            TokenStandard::FungibleAsset | TokenStandard::Fungible => {
-                find_master_edition_account(&nft.mint.pubkey()).0
-            }
-            TokenStandard::NonFungibleEdition | TokenStandard::ProgrammableNonFungibleEdition => {
-                panic!("Invalid test for Editions")
-            }
-        };
+        assert_before_master_edition(&mut context, nft.edition.unwrap()).await;
```

Comment:
wdyt about leaving this here, and changing the test from `cannot_resize_nonfungible_as_update_authority` to just plain `cannot_resize_as_update_authority` and run all four standards?